documentation/app/styles/app.scss

```
@@ -0,0 +1,28 @@
+/**
+* Copyright (c) HashiCorp, Inc.
+*/
+
+@import "@hashicorp/consul-ui-toolkit";
+@import "@hashicorp/design-system-components";
```

Typically, UI kits built on top of other UI kits are imported after, not before (to avoid specificity battles). Can you try and see if this works too?
```suggestion
@import "@hashicorp/design-system-components";
@import "@hashicorp/consul-ui-toolkit";
```